code/src/WijmoProvider/Features/Rows.ts

```diff
@@ -266,6 +267,16 @@ namespace WijmoProvider.Feature {
             ) as CssClassInfo;
         }
 
+        public getRowData(rowNumber: number): string {
```

Comment:
I would make it different. This method might be useful for other use cases and in my opinion, it should not return a stringified JSON but instead the dataItem. Then, in the GridAPI you can use the stringify whenever you need it to be a string.

Reply:
makes sense.

Reply:
Changed it